presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/TestMongoIntegrationSmokeTest.java

```diff
@@ -340,6 +340,18 @@ public void testDropTable()
         assertQueryFails("SELECT * FROM test.drop_table", ".*Table 'mongodb.test.drop_table' does not exist");
     }
 
+    @Test
+    public void testNullPredicates()
+    {
+        MongoCollection<Document> collection = client.getDatabase("testnull").getCollection("null_predicates");
+        collection.insertOne(new Document(ImmutableMap.of("Name", "abc", "Value", 1)));
```

Comment:
Lowercase filed names (name, value) because this test objective isn't case sensitivity.